glotaran/builtin/models/kinetic_spectrum/spectral_shape.py

```diff
@@ -31,10 +31,9 @@ def calculate(self, axis: np.ndarray) -> np.ndarray:
         shape: numpy.ndarray
 
         """
-        matrix = self.amplitude * np.exp(
+        return self.amplitude * np.exp(
             -np.log(2) * np.square(2 * (axis - self.location) / self.width)
         )
-        return matrix
```

Comment:
It returns a matrix with amplitudes representing a spectral shape (like the 'shape' of a gaussian), not a array.shape (n,m). docstring could be clarified that shape refers to the functional form of the amplitudes in the returned object, rather than the type of object.

Reply:
Created issue #446 to remind us to change the wording in the documentation